docs/content/intro/_index.md

```diff
@@ -11,16 +11,12 @@ weight: 10
 
 # Getting Started
 
-New to Brigade or to CI systems in general? Well, you came to the right place: read this material to quickly get up and running.
+New to Brigade or looking to quickly get up to speed with v2? Well, you came to the right place: read this material to quickly get up and running.
 
 ## Table of Contents
 
-- [Brigade at a glance](overview)
-- [QuickStart](quickstart)  
+- What's new in v2 (TODO)
```

Comment:
Added as a possible intro section; not necessarily adamant on it.  Also open to other suggestions for entries here...

Reply:
(Removed; we can always add at a later date.)